tcms/kiwi_auth/tests/test_views.py

```diff
@@ -201,11 +201,10 @@ def test_register_user_by_email_confirmation(self, send_mail):
         )
         expected_body = (
             _(
-                """Welcome %(user)s,
-thank you for signing up for an %(site_domain)s account!
-
-To activate your account, click this link:
-%(confirm_url)s"""
+                "Welcome %(user)s,\n"
+                "thank you for signing up for an %(site_domain)s account!\n\n"
+                "To activate your account, click this link:\n"
+                "%(confirm_url)s"
```

Comment:
I did it in starting to use `override` method but since it is removed now, should I revert it back too. I personally found this way of writing a lot better and readable than using multiline strings

Reply:
Yes, please revert. Regardless of personal style preferences this constitutes an unrelated change and should not be here.

Reply:
:+1: